TestGitHubProj/Services/FirebaseService.swift

```diff
@@ -69,6 +71,35 @@ final class FirebaseService {
         }
     }
     
+    /// Get data from collection "Users"
+    class func readUsersFromDB(collectionName: String, callback: @escaping ([User]) -> Void) -> [User] {
```

Comment:
Looks like readUsersFromDB will always return empty array. Closure `callback` is responsible of dealing with users. Maybe we don't need to have return value. What do you think?

Reply:
@oleksiikolomiietssnapp fixed